rocketpy/rocket/rocket.py

```diff
@@ -292,6 +292,29 @@ def __init__(  # pylint: disable=too-many-statements
         self.I_12_without_motor = inertia[3]
         self.I_13_without_motor = inertia[4]
         self.I_23_without_motor = inertia[5]
+        inertia_matrix = Matrix(
+            [
+                [
+                    self.I_11_without_motor,
+                    self.I_12_without_motor,
+                    self.I_13_without_motor,
+                ],
+                [
+                    self.I_12_without_motor,
+                    self.I_22_without_motor,
+                    self.I_23_without_motor,
+                ],
+                [
+                    self.I_13_without_motor,
+                    self.I_23_without_motor,
+                    self.I_33_without_motor,
+                ],
+            ]
+        )  # Initial Inertia Tensor determinant singularity check
+        if abs(inertia_matrix) == 0:
+            raise ValueError(
+                "The rocket inertia tensor is singular (determinant is zero). "
+            )
```

Comment:
I think having the dry inertia as zero should not stop the simulation, since someone might only have the inertias on the motor for some reason.

In any case, we need the determinant of the Rocket + Motor Inertia Matrix to be different from zero. Could you move this check so that it is done when a motor is added?